strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/util/PypiPackageInfoValidator.java

```diff
@@ -0,0 +1,20 @@
+package org.carlspring.strongbox.util;
+
+import org.carlspring.strongbox.domain.PypiPackageInfo;
+import org.springframework.stereotype.Component;
+
+import javax.validation.*;
+import java.util.Set;
+
+@Component
+public class PypiPackageInfoValidator {
```

Comment:
Cosmetic: Could you please, re-indent according to our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks! :)